packages/ses/NEWS.md

```diff
@@ -1,6 +1,6 @@
 User-visible changes in SES:
 
-# Next release
+# v1.5.0 (2024-05-06)
```

Comment:
Some of these "Next release" are changed to "v1.5.0" as here, but some are changed to "v1.4.0". Are those mistakes?

Reply:
These are not mistakes. All packages rev separately.

Reply:
Good. Resolving.